clients/fides-js/src/components/tcf/TcfPurposes.tsx

```diff
@@ -114,49 +116,40 @@ const PurposeBlock = ({
 };
 
 const TcfPurposes = ({
-  allPurposes,
+  allPurposesConsent,
   allSpecialPurposes,
-  enabledPurposeIds,
+  enabledPurposeConsentIds,
   enabledSpecialPurposeIds,
   onChange,
 }: {
-  allPurposes: PrivacyExperience["tcf_purposes"];
+  allPurposesConsent: TCFPurposeConsentRecord[] | undefined;
+  enabledPurposeConsentIds: string[];
   allSpecialPurposes: PrivacyExperience["tcf_special_purposes"];
-  enabledPurposeIds: string[];
+  // TODO: support legint toggle (fides#4210)
+  // allPurposesLegint: TCFPurposeLegitimateInterestsRecord[] | undefined;
+  // enabledPurposeLegintIds: string[];
   enabledSpecialPurposeIds: string[];
   onChange: (payload: UpdateEnabledIds) => void;
-}) => {
-  const { filtered, legalBasisFilter, setLegalBasisFilter } =
-    useLegalBasisDropdown({
-      allPurposes,
-      allSpecialPurposes,
-    });
-
-  return (
-    <div>
-      <LegalBasisDropdown
-        selected={legalBasisFilter}
-        onSelect={(basis) => setLegalBasisFilter(basis)}
-      />
-      <PurposeBlock
-        label="Purposes"
-        allPurposes={filtered.purposes as TCFPurposeRecord[]}
-        enabledIds={enabledPurposeIds}
-        onChange={(newEnabledIds) =>
-          onChange({ newEnabledIds, modelType: "purposes" })
-        }
-      />
-      <PurposeBlock
-        label="Special purposes"
-        allPurposes={filtered.specialPurposes as TCFPurposeRecord[]}
-        enabledIds={enabledSpecialPurposeIds}
-        onChange={(newEnabledIds) =>
-          onChange({ newEnabledIds, modelType: "specialPurposes" })
-        }
-        hideToggles
-      />
-    </div>
-  );
-};
+}) => (
+  <div>
+    <PurposeBlock
```

Comment:
Right now we are only rendering consent purposes—legint will follow in https://github.com/ethyca/fides/issues/4210